### PR TITLE
[DOCS] Update zh-docs to v15-rc.1 (with #3240c09)

### PR DESCRIPTION
**Pull request by iamchenxin**
The rest of un-updated zh-docs.

### CODE REVIEW

File: docs/docs/10.4-test-utils.zh-CN.md [hunk 1]
```diff
@@ -0,0 +1,244 @@
...
+ReactComponent findRenderedComponentWithType(
+  ReactComponent tree, function componentClass
+)
+```
+
+类似 `scryRenderedComponentsWithType()` 除了只有一个返回结果，并且要么返回这个结果，要么如果还有其他的匹配项就抛出一个异常。
+
+## Shallow rendering（浅渲染）
+
+钱渲染是一个实验性特性，让你渲染一个组件为 "one level deep" 并且断言渲染方法返回的内容，不用担心子组件的行为，它们没有被实例化或者渲染。这个方式不需要一个 DOM。
```
Comment (jimfb) [inline docs/docs/10.4-test-utils.zh-CN.md]: Isn't "钱" the character for "Money"?  Just wanted to confirm that this doesn't say "Money rendering is an experimental feature"?

Comment (iamchenxin) [inline docs/docs/10.4-test-utils.zh-CN.md]: @jimfb Nope , that is [浅 shallow] not [钱 money], `Shallow` is always translated to `浅` in  mainland of china, like 浅拷贝（shallow copy），浅堆（Shallow Heap）。
:D is there some plan to learn chinese?

Comment (iamchenxin) [inline docs/docs/10.4-test-utils.zh-CN.md]: @jimfb Maybe there should have a plan to host a top-level Glossary page(en to non en) for non English docs. Cause in Chinese Mainland React community， there are a few kind of different glossary for a same English word， this makes  meanings are quite vague。
As someone said:  `One of the key steps in becoming immersed in an ecosystem is learning its vocabulary`.

**Comment by jimfb** [comment]
Ok, this looks good now.  Can you squash into a single commit (`git rebase -i`, and `git push -f`), and then I think we can merge.

**Comment by iamchenxin** [comment]
@jimfb rebased.

**Comment by jimfb** [comment]
Thanks!

**Pull request merged**